README.md

```diff
@@ -0,0 +1,26 @@
+#Crassus page :cactus:
+
+![alt-text](Captura de pantalla de 2020-10-25 22-06-42.png)
```

Comment:
I think this image route isn't working because of the spaces in the name. Please consider renaming it, with a better name and dashes or underscore to separate words.